src/types.ts

```diff
@@ -31,12 +31,14 @@ export interface CommandSpec {
 		};
 	};
 	executeOnCascade?: true;
+	passThrough?: true;
 }
 
 export interface InputObject {
 	command: string;
 	data?: string | number;
-	[index: string]: boolean | string | number | undefined;
```

Comment:
Why was this line changed?

Reply:
Because without the change, TypeScript complains:

<img width="829" alt="Screen Shot 2020-07-27 at 10 23 23 AM" src="https://user-images.githubusercontent.com/10079005/88560049-3ae04400-cff3-11ea-89a7-4d977b841ed9.png">